### ISSUE TITLE
bcmul doesn't calculate the correct values

### ISSUE THREAD

**Issue by jhoff** [kvz/locutus#316]
- [x] Have you checked the guidelines in our [Contributing](https://github.com/kvz/locutus/blob/master/CONTRIBUTING.md) document?

### Description
`bcmul` does not calculate values correctly

### PHP 7.0.13
```
bcmul('2.5', '2.5') // "6"
bcmul('2.5', '2.5', 2) // "6.25"
```

### Javascript
```
bcmul('2.5', '2.5') // "625"
bcmul('2.5', '2.5', 2) // "625."
```

**Comment by jhoff** [comment]
This issue should be fixed in https://github.com/kvz/locutus/pull/317

**Comment by kvz** [comment]
Merged #317! 🎉
